src/utis.ts

```diff
@@ -0,0 +1,27 @@
+const regex = /^v(\d+)\.(\d+)\.(\d+)$/;
```

Comment:
nit: There is a missing L from the name of the file.

Reply:
nice catch